covalent/_results_manager/results_manager.py

```diff
@@ -155,11 +155,11 @@ def _get_result_from_dispatcher(
     Raises:
         MissingLatticeRecordError: If the result is not found.
     """
-    adapter = HTTPAdapter()
+    adapter = HTTPAdapter(max_retries=5)
     http = requests.Session()
     http.mount("http://", adapter)
     url = "http://" + dispatcher + "/api/result/" + dispatch_id
-    response = http.get(url, params={"wait": wait, "status_only": status_only})
+    response = http.get(url, params={"wait": wait, "status_only": status_only}, timeout=1)
```

Comment:
@FyzHsn can you review this change? @poojithurao my is the `get_result` being modified as part of this PR?

Reply:
Ya sure, but I'll merge with develop first

Reply:
K, the timeout is set to 2 after resolving the merge conflicts